### ISSUE TITLE
LanguageTab core module not working

### ISSUE THREAD

**Issue by MichaelvanLaar** [ryancramerdesign/ProcessWire#919]
The LanguageTab core module can’t be found in the backend’s modules list although the file `LanguageTabs.module` is in place. I’m using ProcessWire 2.5.3.

In [issue #653](https://github.com/ryancramerdesign/ProcessWire/issues/653) I read that Ryan removed the tabs for page name fields. But if I interpret his comment right this doesn’t mean that the whole Language Tabs module is disabled.

I found a workaround to make the module visible in the modules list again: Move the file `LanguageTabs.module` from directory `/wire/modules/LanguageSupport` to `/wire/modules`. I have no idea why this fixes the problem. I have not found any hint in any file which indicates that ProcessWire doesn’t like the file `LanguageTabs.module` in the subdirectory `/wire/modules/LanguageSupport`

In addition to that a little bug has to be fixed to make the module work again. The JS and CSS file can’t be found because there is a wrong URL in lines 83 and 84 of the file `LanguageTabs.module`. Instead of `$this->config->urls->LanguageTabs` it must be `$this->config->urls->LanguageSupport`.

**Comment by ryancramerdesign** [comment]
I haven't been able to duplicate this one, but will wait to hear if the issue turns up for anyone else.

**Comment by LostKobrakai** [comment]
I had this issue on a client page as well. The module just disappeared as I put the site to their host. It was no issue for them so I didn't investigate further.

**Comment by MichaelvanLaar** [comment]
I just upgraded the above mentioned site to version 2.5.19. The bug is still there. But even the workaround (= moving the module file) doesn’t work anymore. Also the additional installation of the Language Field Tabs from the module directory doesn’t change anything. (I thought it could be worth a try.)

**Comment by ryancramerdesign** [comment]
Definitely uninstall the LanguageTabsFields from the directory, as the two could certainly conflict with one another. I don't know what the issue is and have still not been able to duplicate it, but I did make the dependencies slightly simpler on the LanguageTabs module (requiring LanguageSupport rather than LanguageSupportFields), just in case that was a potential issue. I'm curious to know if you continue to see the issue in 2.5.20+ after refreshing modules.

**Comment by MichaelvanLaar** [comment]
Problem solved. After upgrading to 2.5.20 there was no change, even after refreshing modules. So I opened the database and manually deleted everything referencing LanguageTabs. After that the module showed up again in the core modules tab. I installed it and everything’s allright.

I don’t know if it was the update or the database cleanup which solved the problem. Perhaps the combination.

Anyway, thank you Ryan for your kind help :-)
